app/cli/src/main/scala/org/bitcoins/cli/Cli.scala

```diff
@@ -70,6 +75,41 @@ object Cli extends App {
         .hidden()
         .action((_, conf) => conf.copy(command = GetBestBlockHash))
         .text(s"Get the best block hash"),
+      cmd("rescan")
```

Comment:
Can we define a `help` message on this? https://github.com/scopt/scopt#help-version-and-notes